static/scss/detail/learning-outcomes.scss

```diff
@@ -0,0 +1,56 @@
+// sass-lint:disable mixins-before-declarations
+
+.learning-outcomes-block {
+  background: white url('#{$static-path}/images/bg-access.png') repeat-x 0 0;
+  padding: 50px 0;
+
+  .heading-container {
+    margin: 0 auto;
+    max-width: 1000px;
+    text-align: center;
+    font-size: 28px;
+    line-height: 38px;
+    font-weight: 500;
+  }
+
+  h1 {
+    max-width: 700px;
+    margin: 0 auto 20px;
+    font-size: 40px;
+    line-height: 44px;
+    color: $red;
+    font-weight: 600;
+  }
+
+  p {
+    margin: 0 0 30px;
+  }
+
+  .learning-outcomes-list {
+    margin: 0 -45px;
+    padding: 0;
+    list-style: none;
+    letter-spacing: -.32em;
+    overflow: hidden;
+
+    li {
+      letter-spacing: normal;
+      padding: 15px 0;
+      font-size: 24px;
+      line-height: 28px;
+      color: #8a8b8c;
```

Comment:
`$tile-color-gray` defines this same hex value. Can you use that here and in the line below? While you're at it, there are two spaces before the hex value in the line below